sklearn/preprocessing/value_dropper.py

```diff
@@ -0,0 +1,216 @@
+# Author : Raghav RV <rvraghav93@gmail.com>
+#
+# Licence : BSD 3 clause
+
+import numpy as np
+
+from ..utils import check_X_y
+from ..utils import check_random_state
+
+__all__ = ["drop_values"]
+
+
+def drop_values(X, y=None, missing_mask=None,
```

Comment:
This function seems very complex and very special purpose. Did you have an application? The ``missing_mask_only`` seems a pretty redundant parameter. Also, if I want data MCAR why do I need a function for that?
mask = np.random.uniform() < missing_fraction